guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -73,6 +90,17 @@ ifndef::katello,satellite,orcharhino[]
 --user "_My_User_Name_" \
 -H 'Content-Type: application/json'
 ----
++
+If the above host registration command fails, you may require to add `--insecure` flag.
+It will then allow hosts to get registered with {Project} via {SmartProxy}.
```

Comment:
Can we use the same instruction as for the command line, with the exact instruction replaced?
```suggestion
If your hosts do not trust the SSL certificate of {ProjectServer}, SSL validation can be disabled by adding the `insecure` flag to the registration command.
```

Reply:
Added to CLI method as well :+1: